docs/Inspection-of-yum-Repositories.md

```diff
@@ -199,6 +201,23 @@ Attributes of yum repos are mapped to zypp's attributes.
 * Disadvantages:
   * Building Red Hat images not straight forward: We can find out that it is a yum repo because it doesn't have priority nor autorefresh
 
+
+## Use Cases
```

Comment:
I think the use cases should be at the beginning of the document and define the goal we want to achieve with the implementation.